mewarpx/mewarpx/assemblies.py

```diff
@@ -474,26 +476,33 @@ def __init__(self, V, T, WF, name, r_outer, r_inner=0.0, zmin=None,
 
     def isinside(self, X, Y, Z, aura=0):
         """
-        Calculates which grid tiles are within the cylinder.
+        Determines whether the given coordinates are inside the assembly.
```

Comment:
Shouldn't it still be within the cylinder, since the method is part of the `CylinderZ` class?

Reply:
Assembly is just more general. Next time we make a new assembly we can just copy that doc-string and it will still be accurate.